src/bin/travis/resources/papi_conformance_expected_failures.txt

```diff
@@ -0,0 +1,114 @@
+1
```

Comment:
TOL I probably prefer a single file ATM though that scheme certainly wouldn't scale well to N > 2

Reply:
Open to other ideas. I thought about using a single multi-column file for example. But eventually / in my dream world these files will not be used at all, especially as cwl-test-numbers are NOT stable.